uploader/uploader.py

```diff
@@ -40,6 +40,15 @@ def __init__(self):
                 raise RuntimeError('Won\'t let you upload to localhost because I '
                                    'remove files after uploading them, and you '
                                    'might be uploading to the same directory')
+
+        if not self.url.endswith('/'):
+            print('BAD RSYNC URL')
+            raise RuntimeError(''
+                               'Woa there! It looks like you are trying to '
+                               'use an RSYNC URL ending with a character - '
+                               'All rsync addresses should end / - aborting')
+
+        if self.url.endswith('/'):
```

Comment:
This check is redundant. Due to `raise RuntimeError`, the execution will never reach this point if `self.url` doesn't end with a slash.